src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -536,4 +539,20 @@
      */
     List<BannerPatternShape> getBannerPatternShapes();
 
+    /**
+     * Retrieves the ItemDictionary for this GameRegistry
```

Comment:
Hey don't worry about it, I didn't do much more than look over the code I didn't alter, I was anticipating as you were that stuff would fall out of the review process anyway and that any dumb mistakes, glaring omissions or just minor cruft would be caught in the filter.